packages/workflow/src/workflow.ts

```diff
@@ -81,6 +84,27 @@ export function addDefaultWorkflowOptions<T extends Workflow>(
   };
 }
 
+function addUserMetadata(userMetadata?: UserMetadata): temporal.api.sdk.v1.IUserMetadata | undefined {
+  if (userMetadata == null) {
+    return undefined;
+  }
+
+  const jsonConverter = new JsonPayloadConverter();
```

Comment:
Don't create a JSON payload converter. Use the payload converter from the activator.

Reply:
yup, done